mountpoint-s3/src/fs.rs

```diff
@@ -591,6 +592,13 @@ where
             }
         }
     }
+
+    pub async fn unlink(&self, parent_ino: InodeNo, name: &OsStr) -> Result<(), libc::c_int> {
+        self.superblock
+            .unlink(&self.client, parent_ino, name)
+            .map_err(Into::into)
+            .await
```

Comment:
Is the `map_err` needed?

We could do something like this instead (and remove the `use` above):
```suggestion
        self.superblock.unlink(&self.client, parent_ino, name).await?;
        Ok(())
```

Reply:
I see, yeah we can do either. I'll use `?` just to be consistent.